zaplib/web/wasm_runtime.ts

```diff
@@ -811,7 +813,7 @@ export const initialize: Initialize = (initParams) => {
       });
     };
 
-    if (document.readyState !== "loading") {
+    if (typeof document === "undefined" || document.readyState !== "loading") {
```

Comment:
Not `globalThis.document`?

Reply:
```suggestion
    if (globalThis.document || document.readyState !== "loading") {
```

is indeed a bit more succinct and equivalent.